.github/workflows/pr_check.yaml

```diff
@@ -7,12 +7,20 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        cypress-version: [^10, ^11, ^12]
```

Comment:
Great idea to test multiple Cypress versions. However, this will name the build jobs like `build (^12)`. Let's give them a more readable name like:

```yaml
name: build (Cypress ${{ matrix.cypress-version }})
```

More info: https://futurestud.io/tutorials/github-actions-customize-the-job-name